src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -526,10 +525,10 @@
 	gap: 6px;
 }
 
+.interactive-response-progress-tree,
 .interactive-session .chat-used-context-list {
 	border: 1px solid var(--vscode-chat-requestBorder);
 	border-radius: 4px;
-	padding: 4px;
```

Comment:
This padding was intentional fwiw. It did cause some side effects for lists that scrolled though which I do want to fix. 

Without it, we get this:
![CleanShot 2023-11-01 at 14 11 27@2x](https://github.com/microsoft/vscode/assets/25163139/60b42b8f-0963-4ad6-b9fa-496070121be0)